src/pages/search-results.js

```diff
@@ -0,0 +1,272 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import {
+  Icon,
+  Button,
+  Link,
+  search,
+  Spinner,
+  Surface,
+  useLocale,
+  useQueryParams,
+} from '@newrelic/gatsby-theme-newrelic';
+
+import { usePagination, DOTS } from '../hooks/usePagination';
+
+const SearchResultPageView = ({ pageContext }) => {
+  const { queryParams } = useQueryParams();
+  const query = queryParams.get('query');
+  const page = Number(queryParams.get('page'));
+  const locale = useLocale();
+  const [results, setResults] = useState({});
+  const { records, pageCount } = results;
+  const { slug } = pageContext;
+
+  const totalPages = pageCount;
+  const totalResults = totalPages * 5;
+  const prevPage = page - 1;
+  const nextPage = page + 1;
+  const hasNextPage = nextPage <= totalPages;
+  const hasPrevPage = prevPage >= 1;
+
+  const paginationRange = usePagination({
+    totalPageCount: totalPages,
+    siblingCount: 1,
+    currentPage: page,
+  });
+
+  useEffect(() => {
+    (async () => {
+      const defaultSources = locale.isDefault
+        ? ['developer', 'docs', 'opensource', 'quickstarts']
+        : [
+            `developer-${locale.locale}`,
+            `docs-${locale.locale}`,
+            `opensource-${locale.locale}`,
+            `quickstarts`,
+          ];
+      const results = await search({
```

Comment:
we should handle the error case here. probably just an error message that says something like "unable to get search results, an error has occurred" is fine

Reply:
good point!